app/services/compute_resource_host_importer.rb

```diff
@@ -10,7 +10,7 @@ def initialize(opts = {})
     self.host = Host::Managed.new(
       :managed => true,
       :build => false,
-      :compute_resource_id => compute_resource.id,
+      :compute_resource => compute_resource,
```

Comment:
odd, why didn't it work with the id?

Reply:
I didn't check.

@kamils-iRonin, @leewaa, @justahero: Can anyone of you do me a favour and check this? I think the tests make the desired behaviour clear.

Reply:
@leewaa: Yeah, but shouldn't both approaches work?

Reply:
Yes both approaches should work.

Reply:
The id method does not work in the tests since the `compute_resource` is not persisted to the database.